README.md

```diff
@@ -1,7 +1,7 @@
 9wm
 ====
 
-9wm is an X11 window manager inspired by the Plan 9 window manager 8½.
+9wm is an X11 window manager inspired by the Plan 9 window manager rio.
```

Comment:
The 8½ wording was in the upstream source. I don't think rio existed when dhog wrote 9wm.